production_configs/20220301_psf_tuned_zen20/lstchain_config.json

```diff
@@ -0,0 +1,275 @@
+      "leakage_intensity_width_2"
+    ],
+  
+    "disp_classification_features": [
+      "log_intensity",
+      "width",
+      "length",
+      "wl",
+      "skewness",
+      "kurtosis",
+      "time_gradient",
+      "leakage_intensity_width_2"
+    ],
+  
+    "particle_classification_features": [
+      "log_intensity",
+      "width",
+      "length",
+      "x",
+      "y",
+      "wl",
+      "skewness",
+      "kurtosis",
+      "time_gradient",
+      "leakage_intensity_width_2",
+      "log_reco_energy",
+      "reco_disp_norm",
+      "reco_disp_sign"
+    ],
+  
+    "allowed_tels": [1],
+    "write_pe_image": false,
+    "mc_image_scaling_factor": 1,
+    "image_extractor": "LocalPeakWindowSum",
+    "image_extractor_for_muons": "GlobalPeakWindowSum",
+    "CameraCalibrator": {
+      "apply_waveform_time_shift": false
+    },
+    "time_sampling_correction_path": "default",
+    "LocalPeakWindowSum":{
+      "window_shift": 4,
+      "window_width": 8,
+      "apply_integration_correction": true
+    },
+    "GlobalPeakWindowSum":{
+      "window_shift": 4,
+      "window_width": 8,
+      "apply_integration_correction": false
+    },
+    "timestamps_pointing":"ucts",
+  
+    "train_gamma_src_r_deg": [0, Infinity],
+  
+    "source_dependent": false,
+    "mc_nominal_source_x_deg": 0.4,
+    "mc_nominal_source_y_deg": 0.0,
+  
+    "volume_reducer":{
+      "algorithm": null,
+      "parameters": {
+      }
+    },
+    "calibration_product": "LSTCalibrationCalculator",
+  
+    "LSTCalibrationCalculator":{
+      "systematic_correction_path": null,
+      "squared_excess_noise_factor": 1.222,
+      "flatfield_product": "FlasherFlatFieldCalculator",
+      "pedestal_product": "PedestalIntegrator",
+      "PedestalIntegrator":{
+        "sample_size": 10000,
+        "sample_duration":100000,
+        "tel_id":1,
+        "time_sampling_correction_path": null,
+        "charge_median_cut_outliers": [-10,10],
+        "charge_std_cut_outliers": [-10,10],
+        "charge_product":"FixedWindowSum",
+        "FixedWindowSum":{
+          "window_shift": 6,
+          "window_width":12,
+          "peak_index": 18,
+          "apply_integration_correction": false
+        }
+      },
+      "FlasherFlatFieldCalculator":{
+        "sample_size": 10000,
+        "sample_duration":100000,
+        "tel_id":1,
+        "time_sampling_correction_path": null,
+        "charge_product":"LocalPeakWindowSum",
+        "charge_median_cut_outliers": [-0.5,0.5],
+        "charge_std_cut_outliers": [-10,10],
+        "time_cut_outliers": [2,38],
+        "LocalPeakWindowSum":{
+          "window_shift": 5,
+          "window_width":12,
+          "apply_integration_correction": false
+        }
+      }
+    },
+    "waveform_nsb_tuning":{
+      "nsb_tuning": false,
+      "nsb_tuning_ratio": 0.52,
+      "spe_location": "lstchain/data/SinglePhE_ResponseInPhE_expo2Gaus.dat"
+    },
+    "image_modifier": {
+      "increase_nsb": false,
```

Comment:
This skips the nsb tuning, but I would still set the values to 0 to avoid confusion.